mesonbuild/interpreter.py

```diff
@@ -980,8 +981,11 @@ def first_supported_argument_method(self, args, kwargs):
         mlog.log('First supported argument:', mlog.red('None'))
         return []
 
-class ModuleState:
-    pass
+ModuleState = namedtuple('ModuleState', [
```

Comment:
RIght just past by that part of the code and thought it was worth making that change :)